package.json

```diff
@@ -1,79 +1,79 @@
-        "ng-packagr": "^11.0.0",
-        "protractor": "~7.0.0",
-        "standard-version": "~9.1.0",
-        "ts-node": "~8.3.0",
-        "tslint": "~6.1.0",
-        "typescript": "~4.0.2"
-    },
-    "config": {
-        "commitizen": {
-            "path": "./node_modules/cz-conventional-changelog"
-        }
-    },
-    "husky": {
-        "hooks": {
-            "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
-        }
-    },
-    "version": "0.2.0"
+  "name": "angular-grid-layout-src",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve --port 1234",
+    "start:public": "ng serve --host 0.0.0.0 --port 1234",
+    "build": "ng build",
+    "build:demo-app": "ng build demo-app",
+    "build:demo-app:prod": "ng build demo-app --configuration production",
+    "build:grid": "ng build angular-grid-layout",
+    "build:grid:watch": "ng build angular-grid-layout --watch",
+    "build:grid:prod": "ng build --configuration production angular-grid-layout",
+    "serve:demo-app": "http-server -p 1234 --proxy http://localhost:1234? -c-1 dist/demo-app",
+    "deploy:demo-app": "ng deploy --base-href=/angular-grid-layout/",
+    "test": "ng test angular-grid-layout",
+    "lint": "ng lint",
+    "lint:build": "tsc -p tsconfig.lint.json",
+    "e2e": "ng e2e",
+    "copy-readme": "ts-node --project scripts/tsconfig.scripts.json scripts/copy-file.ts README.md dist/angular-grid-layout/README.md",
+    "commit": "git-cz",
+    "release": "standard-version"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~12.2.14",
+    "@angular/cdk": "~12.2.13",
+    "@angular/common": "~12.2.14",
+    "@angular/compiler": "~12.2.14",
+    "@angular/core": "~12.2.14",
+    "@angular/forms": "~12.2.14",
+    "@angular/material": "~12.2.13",
+    "@angular/platform-browser": "~12.2.14",
+    "@angular/platform-browser-dynamic": "~12.2.14",
+    "@angular/router": "~12.2.14",
+    "@swimlane/ngx-charts": "~17.0.0",
+    "rxjs": "~6.6.0",
+    "tslib": "^2.0.0",
+    "zone.js": "~0.11.4"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~12.2.14",
+    "@angular/cli": "~12.2.14",
+    "@angular/compiler-cli": "~12.2.14",
+    "@commitlint/cli": "~11.0.0",
+    "@commitlint/config-angular": "~11.0.0",
+    "@commitlint/config-conventional": "~11.0.0",
+    "@types/jasmine": "~3.6.0",
+    "@types/node": "^12.11.1",
+    "angular-cli-ghpages": "~1.0.0-rc.1",
+    "codelyzer": "^6.0.0",
+    "cz-conventional-changelog": "~3.3.0",
+    "http-server": "~0.12.3",
+    "husky": "~4.3.8",
+    "jasmine-core": "~3.8.0",
+    "jasmine-spec-reporter": "~5.0.0",
+    "karma": "~6.3.9",
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-coverage": "~2.0.3",
+    "karma-jasmine": "~4.0.0",
+    "karma-jasmine-html-reporter": "^1.5.0",
+    "ng-packagr": "^12.2.6",
+    "protractor": "~7.0.0",
+    "standard-version": "~9.1.0",
+    "ts-node": "~8.3.0",
+    "tslint": "~6.1.0",
+    "typescript": "^4.2.4"
```

Comment:
Better replace '^' for '~'. We don't want new minor version to be installed, only patches.